### DISCUSSION TITLE
S3_sink region

### DISCUSSION THREAD

**Discussion by LinTechSo** [vectordotdev/vector#13053] [Q&A] [answered]
Hi, 
I going to use my local Minio s3 server 
what endpoint should I use for my vector s3 sink region?
can I use "default" or my s3 URI for the region?

**Comment by jszwedko** [comment, marked as answer]
Hi @LinTechSo ,

For `endpoint` you should configure it to point to your Minio server. For `region` you can use any value. I believe Minio will accept it.
